Scripts/Miscellaneous/Spotify_Playlist_Generator/README.md

```diff
@@ -0,0 +1,25 @@
+# Script Title
+<!--Remove the below lines and add yours -->
+Spotify Playlist Generator
+
+### Prerequisites
+<!--Remove the below lines and add yours -->
+1. Spotipy 
+` pip install spotipy --upgrade `
+
+### How to run the script
+<!--Remove the below lines and add yours -->
+1. [Create Spotify Developers Account and create a new app](https://developer.spotify.com/)
+2. Export Client Id, Client Secret and Redirect URI
+    * `export SPOTIPY_CLIENT_ID='your-spotify-client-id'`
+    * `export SPOTIPY_CLIENT_SECRET='your-spotify-client-secret'`
+    * `export SPOTIPY_REDIRECT_URI='your-app-redirect-url'`
+3. `python main.py -p [playlist_id] -l [size_of_each_playlist]`
```

Comment:
Hi! Can you add a test id and playlist? For viewers to test.

Reply:
Hi! I don't have a test account on Spotify, i tested these on my personal account.